src/dmd/chkformat.d

```diff
@@ -915,6 +942,26 @@ Format parseGenericFormatSpecifier(scope const char[] format,
             break;
     }
 
+    if (findCondition(global.versionids, Identifier.idPool("CRuntime_Glibc")))
+    {
+        // GNU extensions
+        // printf: http://www.gnu.org/software/libc/manual/html_node/Other-Output-Conversions.html
+        // scanf: https://www.gnu.org/software/libc/manual/html_node/Dynamic-String-Input.html
+        switch (sc)
+        {
+            case 'a':
```

Comment:
It seems that `a` is already handled up there ? Maybe just merge the switches and have a `findCondition` in there.